docs/API.md

```diff
@@ -1,33 +1,59 @@
+# SRT API
+
+The SRT C API (defined in `srt.h` file) is largely based in design on the legacy 
+UDT API, with some important changes. The `udt.h` file contains the legacy UDT API 
+plus some minor optional functions that require the C++ standard library to be used. 
+There are a few optional C++ API functions stored there, as there is no real C++ API 
+for SRT. These functions may be useful in certain situations.
+
+There are some example applications so that you can see how the API is being used, 
+including `srt-live-transmit`, `srt-file-transmit` and `srt-multiplex`. All SRT 
+related material is contained in `transmitmedia.*` files in the `common` directory 
```

Comment:
Things have changed over time here and it wasn't updated:

1. `srt-multiplex` no longer exists. The architecture was updated in `srt-live-transmit` and therefore this application has been moved to test, as only they preserve the old architecture. So this file should be removed from this list.
2. The directory that contains these `transmitmedia.*` files is now `apps`, not `common`.

Rest of the description can stay as is.